src/internal/Popover.js

```diff
@@ -281,6 +295,17 @@ class Popover extends Component<DefaultProps, Props, void> {
     }
   };
 
+  handleResize = debounce(
+    () => {
+      const element: any = ReactDOM.findDOMNode(this.transitionEl);
+      this.setPositioningStyles(element);
+    },
+    166,
+    {
+      leading: true,
```

Comment:
What about the last value with leading: true. Do we have it? I feels like we don't. https://sking7.github.io/articles/1248932487.html